core/src/main/kotlin/org/evomaster/core/problem/rest/service/AIResponseClassifier.kt

```diff
@@ -11,46 +11,38 @@ import org.evomaster.core.problem.rest.classifier.GaussianOnlineClassifier
 import javax.annotation.PostConstruct
 
 
+
 class AIResponseClassifier : AIModel {
 
     @Inject
-    private lateinit var config : EMConfig
+    private lateinit var config: EMConfig
 
-    private lateinit var delegate: AIModel
+    private lateinit var model: AIModel
```

Comment:
why changing the name from `delegate` to `model`?  `AIResponseClassifier` itself extends `AIModel`, but delegates its execution to another model.